### ISSUE TITLE
Missing end quote adds line break rather than adding missing quote

### ISSUE THREAD

**Issue by joshsummers4** [josdejong/jsonrepair#97]
If I accidently remove a quotation mark from the end of a value (like at the end of Fiber on line 1.
```
{
  "customerType": "Prepaid/Postpaid/Fiber,
  "emailAddress": "eample@domain.com",
  "isknown": "1",
  "phoneNumber": "123456"
}
```

The errow message is 

> Bad control character in string literal in JSON at line 2 column 0

The auto repair will add a line break and wrap around the next quotation mark rather than just add the missing quotation as would seem like the sensible fix. So we end up with the following:

```
{
  "customerType": "Prepaid/Postpaid/Fiber,\n  ","emailAddress":": ","eample@domain.com":",\n  ","isknown":": ","1":",\n  ","phoneNumber":": ","123456":"\n}"}
```

Is there a way to fix this issue alternatively is there a way to turn off the auto repair function for certain/all errors but still show the user there is an error?

**Comment by josdejong** [comment]
🤔 hmm I have to think that through. Right now `jsonrepair` assumes that a text can be multiline, but it may make more sense in general to not assume multiline text. I think we'll have to do some fiddling with that. Help improving the library for this case would be welcome.

**Comment by josdejong** [comment]
This has been fixed now in `v3.2.1`
